plugins/commands/login/plugin.rb

```diff
@@ -12,9 +12,10 @@ class Plugin < Vagrant.plugin("2")
       DESC
 
       command(:login) do
-        require_relative "command"
+        require File.expand_path("../../cloud/auth/login", __FILE__)
         init!
-        Command
+        # TODO: Print dep warning here?
```

Comment:
Yeah, it would be good to print a deprecation warning so we can just remove this command on a future release.